TextMate/TextMate2.munki.recipe

```diff
@@ -23,6 +23,8 @@ preferences panel.
         <string>TextMate2</string>
         <key>BRANCH</key>
         <string>release</string>
+        <key>MACOS</key>
```

Comment:
Same here - the TextMate changes should be in a separate PR on master.

Reply:
Done. TextMate changes now in separate PR #318